.github/workflows/update.yml

```diff
@@ -0,0 +1,30 @@
+name: Update Subreddit CSS
+
+on:
+  push:
+    paths:
+      - 'src/main.css'
```

Comment:
Question:

What's the behavior here? Ideally, the behavior we need:

- only trigger when a pull request is opened against main branch AND main.css file has changed

Let me try the script locally once as well

Reply:
Update, only trigger when pull request is successfully merged